locations/spiders/campingworld.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import scrapy
+from scrapy.http import JsonRequest, Request
+
+import base64
+
+from locations.items import Feature
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class CampingWorldSpider(scrapy.Spider):
+    name = "campingworld"
+    item_attributes = {"brand": "Camping World", "brand_wikidata": "Q5028383"}
+
+    locator_url = "https://rv.campingworld.com/locations"
+    api_url = "https://api.rvs.com/api/geodata/getclosestdealer"
+    user_agent = BROWSER_DEFAULT
+
+    request_json = """
+        {
+            "zipCode":"66952",
+            "domainId":1,
+            "take":2000,
+            "hours":true,
+            "facilities":true,
+            "requirements":{},
+            "glCodes":"",
+            "standAlone":true
+        }
+    """.strip()
+
+    def start_requests(self):
+        yield Request(url=self.locator_url, callback=self.parse_locator)
+
+    def parse_locator(self, response):
+        # The locator HTML contains a JWT in a meta tag that we need to make the API call.
+        meta = response.xpath('//meta/@content')
+        meta_content = [s.get() for s in meta]
+        tokens = [s for s in meta_content if s.startswith("eyJ") and ".eyJ" in s]
+        token = tokens[0]
```

Comment:
Would it be better to do this in the xpath?
```suggestion
        token = response.xpath('//meta[starts-with(@content, "eyJ")][contains(@content, ".eyJ")]/@content').get()
```

Reply:
Works for me.